internal/ignition/ignition.go

```diff
@@ -1481,9 +1495,18 @@ func (ib *ignitionBuilder) FormatDiscoveryIgnitionFile(ctx context.Context, infr
 		return "", err
 	}
 
-	res := buf.String()
+	res := ""
+	if UseConvergedFlow {
+		res, err = ib.getConvergedDiscoveryTemplate(buf.Bytes())
```

Comment:
i wonder why it should be different from , for example OKD flow? everything is in the same template and in case you need you provide relevant flags? Here if i understand right you create additional configs and append them. It is ok though creates another flow. Maybe we can stick to existing one?

Reply:
It's different because we wrote the OKD services ourselves and we get the Ironic ignition bits from CBO.
in order to generate the Ironic ignition, CBO requires a few parameters that we need to fetch during runtime so it makes more sense to  get the Ironic ignition from CBO and merge it with ours